templates/short/alarm.vm

```diff
@@ -1 +1,2 @@
+#set($title = "Alarm")
```

Comment:
Do all these titles match email titles?

Reply:
In a new commit, copied subjects from "full" templates into "short" templates.